v2/metadata/client.go

```diff
@@ -376,6 +408,10 @@ func (c *Client) RecordArtifact(ctx context.Context, schema string, artifact *pb
 	return getRes.Artifacts[0], nil
 }
 
+func GetIDFromExecution(execution Execution) *int64 {
```

Comment:
Define this as a method of Execution type?
GetID()

Reply:
done